elastic/indexer/insta_users/model.go

```diff
@@ -0,0 +1,8 @@
+package main
+
+type user struct {
```

Comment:
let's export this

Reply:
why? do we need the user outside of the package

Reply:
Let's just do it as soon we need it :)